src/main/java/seedu/address/model/item/VisualizeList.java

```diff
@@ -27,6 +27,14 @@ public boolean add(Item item) {
         }
     }
 
+    public void add(int targetIndex, Item item) {
+        if (hasItem(item)) {
+            return;
+        } else {
+            list.add(targetIndex, item);
+        }
+    }
+
```

Comment:
Do you want to remove this method? There is already an implementation for this method in the parent class of VisualizeList which also triggers a change. This might break the reminder feature 😅

Reply:
Yes, it does break reminder, so I am renaming it